src/main/java/com/comphenix/protocol/reflect/FieldUtils.java

```diff
@@ -153,29 +155,21 @@ public static Field getField(final Class cls, String fieldName, boolean forceAcc
      * @return the <code>List</code> of interfaces in order,
      *  <code>null</code> if null input
      */
-    private static List getAllInterfaces(Class cls) {
-        if (cls == null) {
-            return null;
-        }
-        List<Class> list = new ArrayList<Class>();
-        
-        while (cls != null) {
-            Class[] interfaces = cls.getInterfaces();
-            for (int i = 0; i < interfaces.length; i++) {
-                if (list.contains(interfaces[i]) == false) {
-                    list.add(interfaces[i]);
-                }
-                List superInterfaces = getAllInterfaces(interfaces[i]);
-                for (Iterator it = superInterfaces.iterator(); it.hasNext();) {
-                    Class intface = (Class) it.next();
-                    if (list.contains(intface) == false) {
-                        list.add(intface);
-                    }
-                }
-            }
-            cls = cls.getSuperclass();
-        }
-        return list;
+    private static Set<Class> getAllInterfaces(Class cls) {
+		if (cls == null) return null;
```

Comment:
nitpick, but this looks overindented

Reply:
I checked again locally and this is actually fine. GitHub just displays this wierdly.
![grafik](https://user-images.githubusercontent.com/22870297/171857025-1531f7c8-9749-46d4-82cf-e0f610475326.png)

Reply:
The method itself is also indented correctly.

Reply:
It looks weird because the rewritten code used spaces. The rest of the file uses tabs.